packages/hardhat-zksync-node/src/index.ts

```diff
@@ -308,7 +308,7 @@ task(TASK_NODE_ZKSYNC, 'Starts a JSON-RPC server for ZKsync node')
     .addFlag('enforceBytecodeCompression', 'Enforce bytecode compression')
     .addOptionalParam('systemContractsPath', 'Path to the system contracts', undefined, types.string)
     .addOptionalParam('protocolVersion', 'Protocol version to use for new blocks (default: 26)', undefined, types.int)
-    .addFlag('emulateEvm', 'Emulate EVM')
+    .addFlag('evmInterpreter', 'EVM Interpreter')
```

Comment:
This is a breaking change, right? Shouldn't we make it backward-compatible (e.g. leave both options, but emit a deprecation warning if the old name is used)?

Reply:
+1, especially given that we received a complaint about us making breaking changes in patch/minor releases :grimacing: